src/api/app/policies/assignment_policy.rb

```diff
@@ -9,4 +9,8 @@ def create?
 
     record.assignee_is_a_collaborator
   end
+
+  def destroy?
+    create?
+  end
```

Comment:
We should add policy tests for this new policy...

Reply:
Maybe you overlooked it as it was a small change, it's [down below](https://github.com/openSUSE/open-build-service/pull/17852/files#diff-1f5623e5f760298b7dbfc2273b6800f2376bc2c264fb49cd8f4ade2fdc590d0dR4)